Dockerfile

```diff
@@ -11,4 +11,4 @@ WORKDIR /
 COPY --from=build /go/src/github.com/Invoca/go-flashpaper/go-flashpaper .
 
 EXPOSE 8443
-RUN go-flashpaper
+RUN ./go-flashpaper
```

Comment:
I'd suggesting making this the _exec_ form. You can read more on the difference [here](https://docs.docker.com/engine/reference/builder/#run).